src/app/components/import/ImportGroupsModalContent.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { c } from 'ttag';
+import { Alert, Block, Row, Select } from 'react-components';
+
+const ImportGroupsModalContent = () => {
+    return (
+        <>
+            <Alert>
+                {c('Description')
+                    .t`We picked up at least one group/organization from the list of contacts you are importing. Take the time to review how we should import these groups.`}
+            </Alert>
+            <Block>
+                <Row>
+                    {c('jurrr').t`Family (5 contacts)`}
+                    <Select
+                        value={1}
+                        options={[{ text: 'uno', value: 1 }, { text: 'dos', value: 2 }, { text: 'tres', value: 3 }]}
```

Comment:
These labels need translations no?

Reply:
This component is actually totally dummy at the moment. I will take care of it in a different PR. It's just here not to forget it! ;)